test/test_random.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import pytest
+
+import sharpy as sp
+
+
+@pytest.fixture(params=[(), (6,), (6, 5), (6, 5, 4)])
+def shape(request):
+    return request.param
+
+
+@pytest.fixture(params=[0, 42, 66, 890, 1000])
+def seed(request):
+    return request.param
+
+
+def test_random_rand(shape, seed):
+    sp.random.seed(seed)
+    sp_data = sp.random.rand(*shape)
```

Comment:
This need the device argument (from utils, see other tests)
Same below for uniform.

Reply:
Done.